packages/react-card/Spec.md

```diff
@@ -0,0 +1,403 @@
+# @fluentui/react-card Spec
+
+## Background
+
+A cards main function is to provide the scaffolding for hosting actions and content for a single topic within a card sized object. It is a framework for organizing content within the confines of a card.
+
+The card itself should not have content or actions built in, but provide the mechanisms for it to be displayed.
+
+### Fabric (v8)
+
```

Comment:
There's actually a better previous attempt at a `Card` component in v8 under the [`@fluentui/react-cards`](https://github.com/microsoft/fluentui/tree/master/packages/react-cards) package.

Reply:
I think we can disregard this implementation as it's not publicly used. Unless you mean that this implementation is a better attempt than the `react-card` implementation we had?

Reply:
No, I meant the former, but you are right, it's not publicly used so let's keep it off this file.